core/server/api/shared/frame.js

```diff
@@ -37,6 +37,7 @@ class Frame {
             }
         }
 
+        this.options.rawInclude = apiConfig.rawInclude;
```

Comment:
This field was added to allow preserving options in an array as it came in (same logic in v1 - https://github.com/TryGhost/Ghost/blob/7fafa1e152708a8b672bb6b19d9dacf5680aa2d3/core/server/api/v0.1/utils.js#L286):
```
                if (!convertOptions.forModel) {
                    options.include = utils.prepareInclude(options.include, allowedIncludes);
                } else {
                    options.withRelated = utils.prepareInclude(options.include, allowedIncludes);
                    delete options.include;
                }
```

Was trying to find a better place for this but ended up putting it into options for now. Definitelly would like to discuss better place to put it (or maybe way around it so we don't need it at all)